packages/react/src/contexts/session.context.tsx

```diff
@@ -173,8 +173,8 @@ export function SessionContextProvider({ api, data, children }: SessionContextPr
   }
 
   function sync(): void {
-    const token = localStorage.getItem('dfx.authenticationToken');
-    if (token) setAuthenticationToken(token);
+    const token = getAuthToken();
+    if (token) setAuthToken(token);
   }
```

Comment:
I think this method was used to overwrite the token in the cache with the one from storage. I think this is not working anymore.